src/Oscoin/API/HTTP/Handlers.hs

```diff
@@ -119,15 +120,16 @@ lookupBlocksByHeight = do
     blocks <- liftNode $ Node.lookupBlocksByHeight (start, end)
     respond ok200 $ Ok (Chrono.toOldestFirst blocks)
 
-getStatePath
+getStateValue
     :: (ApiTx c tx)
     => Text
     -> ApiAction c tx s i ()
-getStatePath _chain = do
-    path <- listParam "q"
-    result' <- liftNode $ Node.getPathLatest path
-    case result' of
-        Just val ->
-            respond ok200 (Ok val)
-        Nothing ->
-            respond notFound404 $ errBody "Value not found"
+getStateValue encodedKey =
+    case decodeBase58btc (toS encodedKey) of
```

Comment:
This is a strange API to me... The previous one had the benefit of being human readable. Can't we keep the old API and just encodeUtf8 it to convert it to a bytestring?

Reply:
If we must use some kind of encoding, then I would opt for a simple hex/base16 encoding - there's no reason to trade space for implementation complexity here.

Reply:
The API needs to be deal with arbitrary bytestrings. The OscoinTx WorldState uses ByteString as its key and this ByteString is most likely not valid `utf8`. Spock does not give us a way to extract ByteString values from parameters so we need to use our own encoding. I will switch it to hex, though.